src/client/components/ui/SquareMeter/SquareMeter.test.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
+
+import { SquareMeter } from '~/client/components/ui/SquareMeter/SquareMeter';
+
+describe('<SquareMeter />', () => {
+	it('affiche une abréviation', async () => {
+		render(<SquareMeter />);
+
+		// NOTE (GAFI 06-02-2023): nécessaire puisque texte splitté dans plusieurs balises sans rôle associé
```

Comment:
je ne considère pas que tu ais à justifier d'une note ton test

Reply:
L'idée était d'éviter au suivant de perdre du temps en se disant wtf et en essayant de refacto mais is tu penses que ça alourdit le test pour rien et que c'est compréhensible sans je peux le virer, qu'est-ce que tu en penses ?